Makefile

```
@@ -201,6 +211,251 @@ install-tools:
+	@echo "Running TLC on spec/RaftstoreApplyPublish.tla..."
+	./scripts/tla/tlc.sh spec/RaftstoreApplyPublish.tla
+
+tlc-root-replay-watch:
+	@echo "Running TLC on spec/RootReplayWatch.tla..."
+	./scripts/tla/tlc.sh spec/RootReplayWatch.tla
+
+tlc-fsmeta-namespace:
+	@echo "Running TLC on spec/FSMetaNamespace.tla..."
+	./scripts/tla/tlc.sh spec/FSMetaNamespace.tla
+
+tlc-leaseonly-counterexample:
```

checkmake (0.3.2)

[warning] Target body for "tlc-leaseonly-counterexample" exceeds allowed length of 5 lines (7).

(maxbodylength)